pkgs/package-overrides.nix

```diff
@@ -178,6 +178,19 @@ in
         ourPatches ++ upstreamPatches;
     });
 
+    redis =
+      if lib.versionOlder prev.php.version "7.0" then
+        prev.extensions.redis.overrideAttrs (attrs: {
+          name = "redis-3.1.4";
```

Comment:
Thanks.

Actually, we might want to update still:

https://github.com/phpredis/phpredis/commit/17d30d88a5ba67389e05dd6719cf3bfe8c7eb727#diff-37a67ff78eb7260214b323353263b9af40a2fa98719a1e81937fae0159df87a3L41-R44

Or even to 4.0 since PHP 5 support has only been dropped in 5.0.